### PR TITLE
Fix Apple Search Ad Attribution - AIS-267

### PR DESCRIPTION
**Pull request by E-B-Smith**
Stop sending the Apple search ad data after attribution has been found or 30 days.

### CODE REVIEW

**Comment by BranchMacMini** [comment]
Result of [Integration 1](https://stlt.herokuapp.com/v1/xcs_deeplink/qabot.stage.branch.io/27f4d5714d6b6650fe19a53e12c720fd/27f4d5714d6b6650fe19a53e12c723a5)
---
*Duration*: 4 minutes and 47 seconds
*Result*: **Perfect build!** All 179 tests passed. :+1:
*Test Coverage*: 57%

File: Branch-SDK/Branch-SDK/BNCSystemObserver.h [hunk 1]
```diff
@@ -32,5 +32,6 @@ typedef NS_ENUM(NSInteger, BNCUpdateState) {
 + (void)setUpdateState;
 + (BOOL)isSimulator;
 + (BOOL)adTrackingSafe;
++ (NSDate*) appInstallDate;
```
Comment (ahmednawar) [inline Branch-SDK/Branch-SDK/BNCSystemObserver.h]: Where does this get set?

Comment (E-B-Smith) [inline Branch-SDK/Branch-SDK/BNCSystemObserver.h]: Here:
https://github.com/BranchMetrics/ios-branch-deep-linking/blob/master/Branch-SDK/Branch-SDK/BNCSystemObserver.m#L229-L247

**Comment by ahmednawar** [comment]
Exciting! Thanks for the quick fix 👍

**Pull request merged**